### PR TITLE
Release Notes CIS 2.7.1

### PR DESCRIPTION
**Pull request by vklohiya**
**Description**:  Release notes for CIS 2.7.1

## General Checklist
- [x] Updated Added functionality/ bug fix in release notes

### CODE REVIEW

**Review by sravyap135 — APPROVED** [review]
LGTM

File: docs/RELEASE-NOTES.rst [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+FIC Enhancements
+------------------
+* Added support for FIC installation using Helm Charts, Refer `Documentation <https://github.com/F5Networks/f5-ipam-controller/blob/main/helm-charts/f5-ipam-controller/README.md>`_
+
+Known issues
+````````````
+* CIS does not delete the arp entries immediately from BigIP, When we remove all the endpoints for a service in cccl mode,
+* Unable to pass multiple infoblox labels to FIC helm charts
+* Deletion of EDNS resource not removing Wide IP config from BigIP intermittently
+* CIS sends the failed tenant declaration every 30 secs with filter-tenant parameter when a 422 error occurs in as3 response
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:29]: Upgrade notes:

- Moving to CIS >2.4.1 requires update to RBAC and CR schema definition before upgrade. See <link to cluster role binding>and CR schema.

- Moving from CIS > 2.6 with IPAM, see troubleshooting guide for IPAM issue `ipams.fic.f5.com not found`. Refer <link https://github.com/F5Networks/f5-ipam-controller/pull/93/files#diff-46168f4bc896522980775ba4c620234180c70bfbe9d85e51f69523c92d062adeR104 >

**Review by trinaths — CHANGES_REQUESTED** [review]

File: docs/RELEASE-NOTES.rst (hunk 1 above)
Comment (vklohiya) [reply docs/RELEASE-NOTES.rst:29]: Made the required changes please review it.

File: docs/RELEASE-NOTES.rst [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+
+Upgrade notes
+``````````````
+* Moving from CIS > 2.6 with IPAM, see troubleshooting guide for IPAM issue ipams.fic.f5.com not found. Refer `Troubleshooting Guide <https://github.com/sravyap135/f5-ipam-controller/blob/faq/docs/faq/README.md#troubleshooting>`_
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:32]: can we highlight the issue like
* Moving from CIS > 2.6 with IPAM, see troubleshooting guide for IPAM issue -  `ipams.fic.f5.com not found`.

Comment (trinaths) [reply docs/RELEASE-NOTES.rst:32]: just add " ` " to look like a code style.

**Review by trinaths — CHANGES_REQUESTED** [review]

File: docs/RELEASE-NOTES.rst (hunk 2 above)
Comment (vklohiya) [reply docs/RELEASE-NOTES.rst]: Done

**Review by trinaths — APPROVED** [review]
Cool. approved.

**Pull request merged**